contrib/hawq-hadoop/Makefile

```diff
@@ -41,9 +41,9 @@ all: $(DEFAULTTARGET)
 $(DEFAULTTARGET):
 ifdef MAVEN
 ifeq ($(shell java -version 2>&1 | grep 1.7.0 > /dev/null; printf $$?),0)
-	$(MAVEN) package -DskipTests -Dhttps.protocols=TLSv1.2
+	$(MAVEN) package -DskipTests -Dmaven.javadoc.skip=true  -Dhttps.protocols=TLSv1.2
```

Comment:
Why disable java doc pluggin?

Reply:
this is because the javadoc added doclint in jdk8 which required stricter code format (which is not followed by the former code), so I temporarily disabled the javadoc.

Reply:
what's the reason of update hawq-hadoop/Makefile while this is a commit to add compile suppport for orc ?

Reply:
maybe I'll start a subtask individually for JDK8 support in another commit, thanks for your review!